VAMobile/e2e/tests/Appointments.e2e.ts

```diff
@@ -116,10 +116,10 @@ describe('Appointments Screen', () => {
     await element(by.id('getDateRangeTestID')).tap()
```

Comment:
this is of course not critical, but explicitly spelling out 'Past 3 months' three times in this specific test (lines 118, 120, and 123), when we have a constant in this very script of 'past 3 months', hurts me a little.